package.json

```diff
@@ -38,20 +38,18 @@
     "package-readme-entry": "./scripts/package-readme-entry.sh"
   },
   "devDependencies": {
+    "@babel/cli": "7.0.0",
+    "@babel/core": "7.0.0",
+    "@babel/plugin-proposal-export-default-from": "7.0.0",
+    "@babel/plugin-proposal-export-namespace-from": "7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0",
+    "@babel/preset-env": "7.0.0",
+    "@babel/preset-flow": "7.0.0",
     "@commitlint/cli": "7.1.1",
     "@commitlint/config-conventional": "7.1.1",
-    "babel-cli": "6.26.0",
-    "babel-core": "6.26.3",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
This one is still needed for `babel-jest` below ([ref](https://jestjs.io/docs/en/getting-started#using-babel)).

Reply:
Yup.